packages/compass/src/app/styles/index.less

```diff
@@ -70,6 +70,7 @@ html, body {
 * {
   user-select: text;
   -webkit-user-select: text;
+  outline-color: #019EE2;
```

Comment:
This is an important styling change. With updating electron we now have all inputs showing focused outlines. This updates the shown outline color to the focus color of LeafyGreen (it was a yellow).